package.json

```diff
@@ -45,7 +45,8 @@
     "proto:iam": "pbjs -t static-module -w commonjs -p protos google/iam/v1/iam_policy.proto | pbts -o proto/iam.d.ts -",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "benchwrapper": "node bin/benchwrapper.js"
+    "benchwrapper": "node bin/benchwrapper.js",
+    "license-check": "jsgl --local ."
```

Comment:
I'd rather not add this or the `js-green-licenses` dependency below, unless we have a plan to do this consistently across the board.  If we *do* want to do this, I'd advocate for a bot over a GitHub Action in this case.

Reply:
Ah okay. I can remove it again. I was mostly trying to take the union of all of the GitHub actions I found elsewhere :)

Reply:
At some point we'll converge on a single config 😆

Reply:
I think I'm with @JustinBeckwith on this one, or at least doing the work in a separate PR than this one.

We have our GitHub repos templatized, so once we have a `.github/workflows/ci.yml` that we feel is sufficient, I would actually like to pull it out into our [template directory here](https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/node_library).